decorators/overridable.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { withPropsFromContext } from 'react-context-props'
+
+const overridable = (styles = {}, designName) => (Target) => {
+  const OverridableComponent = React.createClass({
```

Comment:
What should we use instead? We use `React.createClass` for all other components.